src/docfx/lib/LocUtility.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class LocUtility
+    {
+        public static CultureInfo GetCultureInfo(string locale)
+        {
+            Debug.Assert(!string.IsNullOrEmpty(locale));
+
+            try
+            {
+                return new CultureInfo(locale);
+            }
+            catch (CultureNotFoundException)
```

Comment:
>CultureNotFoundException [](start = 19, length = 24)

report error?

Reply:
`locale` is user input, so should report error

---
In reply to: [197371175](https://github.com/dotnet/docfx/pull/2912#discussion_r197371175) [](ancestors = 197371175)